psychopy/visual/roi.py

```diff
@@ -85,19 +85,19 @@ def numLooks(self):
     @property
     def isLookedIn(self):
         """Is this ROI currently being looked at"""
-        # Get current eye position
-        if hasattr(self.tracker, "getPos"):
-            pos = self.tracker.getPos()
-        elif hasattr(self.tracker, "getPosition"):
-            pos = self.tracker.getPosition()
-        else:
-            # If there's no position functions, assume False
-            return False
-        if pos is None:
-            # If there's no eye data (e.g. during a blink) assume False
+        try:
+            # Get current device position
+            pos = self.device.getPos()
```

Comment:
Is there any instance where we'll need the `if hasattr(self.device, "getPosition")` check or was that just because the device called it `getPosition`?

Reply:
I checked and the eye tracker and mouse both have a getPos method already so i think we are good.